website/content/docs/connect/config-entries/service-router.mdx

```diff
@@ -703,6 +708,13 @@ spec:
       type: 'bool: false',
       description: 'Allows for connection failure errors to trigger a retry.',
     },
+    {
+      name: 'RetryOn',
+      type: 'array<string>',
+      description: `Allows retrying requests for a list of conditions. One of:
```

Comment:
```suggestion
      description: `Allows Consul to retry requests when the requests meet a specific set of conditions. You can specify one of the following lists:
```
Is that what this means?

Reply:
Yes. How about this? `Allows Consul to retry requests when they meet one of a set of conditions. The available conditions are:`

Reply:
That's pretty good. We can probably even trim it down a bit more and improve consistency with the style guide:

`Allows Consul to retry requests when they meet one of the following sets of conditions:`

Reply:
Updated